backend/open_webui/models/auths.py

```
@@ -143,32 +143,45 @@ def authenticate_user(self, email: str, password: str) -> Optional[UserModel]:
+    def authenticate_user_by_api_key(
+        self, api_key: str, db: Optional[Session] = None
+    ) -> Optional[UserModel]:
+        log.info(f"authenticate_user_by_api_key")
         # if no api_key, return None
         if not api_key:
             return None
 
         try:
-            user = Users.get_user_by_api_key(api_key)
+            user = Users.get_user_by_api_key(api_key, db=db)
             return user if user else None
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)